src/Dispatcher/MASA.Contrib.Dispatcher.IntegrationEvents.Dapr/IProcessor.cs

```diff
@@ -4,10 +4,5 @@ public interface IProcessor
 {
     Task ExecuteAsync(CancellationToken stoppingToken);
 
-    Task SleepAsync();
-
-    /// <summary>
-    /// The time to rest after the task is executed, in milliseconds
-    /// </summary>
-    int SleepTime { get; }
+    Task DelayAsync(int delay);
```

Comment:
If the parameter name does not specify a unit, add a comment

Reply:
processed

Reply:
Why not milliseconds or `TimeSpan`?

Reply:
This is changed to millseconds, but the default unit of time configured in DispatcherOptions is seconds. I think background tasks can be used in most scenarios in seconds.